README.md

```diff
@@ -1,86 +1,63 @@
-
-The [TBox Generator](./JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/converter) is a tool to generate the TBox of an ontology, i.e. the definition of concepts and relationships within it, from a CSV file. The input is a simple, plain-text, tabular definition of concepts and their relationships. The generated output can be in the form of either a collection of subject-predicate-object triples, to be uploaded directly to a triple store, or an OWL file – an industry-standard XML-based format.
-
-##### Agent template classes
-
-A developer who wants to create their own agents in Java may want to use the classes [here](./JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/agent).
-
-##### Derived Information Framework
-
-The [Derived Information Framework](./JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/derivation) library provides an ontological representation of provenance within a KG. That is, it captures which pieces of information stored in a KG are derived from which others and how (i.e. using which agent). It further enables checking whether a derived piece of information is up-to-date, and if not, can trigger an update of outdated information by calling the agent responsible for deriving the information in the first place. The derivation client that is part of the library manages the ontological mark-up required in the KG as well as the process of triggering the relevant agent and updating the derived information if it has been found to be out-of-date when accessed.
+**TWA Base Library:**<br/>
+The [base lib](./JPS_BASE_LIB) is a collection of functionality that is shared among many parts of the code. Core functions include the ability to generate and upload TBoxes, query KGs and RDBs, implement RESTful APIs, and triple cloning.
 
-##### Time-series client
+**Digital Twin Visualisation Framework:**<br/>
+The [DTVF](./web/digital-twin-vis-framework) is a container that provides a web-page with which a user can view and explore geospatial information held in the TWA ecosystem. This includes not only map data, 3D-representations of buildings, and building information models (BIM), but also any data or information connected to them within the KG, such as data calculated by models, acquired sensor data, or other time series data.
 
-This library facilitates dealing with time series, which are ubiquitous in real-world applications, thus minimising code repetition for developers. Ontological mark-up, in the form of triples in a triple store, is designed to be minimal, just containing basic high-level information such as names and types of quantities of which a time series is meant to be stored. The actual data, i.e. time stamps and associated values of the quantities, are being held in a relational database, thus maintaining favourable execution speed and storage space efficiency. The time-series client can be found [here](./JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/timeseries).
+**Intelligent Agents:**<br/>
```

Comment:
The word "intelligent" in this context is loaded. I would go for something less controversial like "autonomous".